tests/test_template.py

```diff
@@ -142,7 +147,7 @@ def test_concept_refinement_grounding():
     assert spatial_region_ctx.refinement_of(
         spatial_region, refinement_func=is_ontological_child, with_context=True
     )
-    assert spatial_region_ctx.refinement_of(
+    assert not spatial_region_ctx.refinement_of(
```

Comment:
I don't think I understand this one: `spatial_region_ctx` is the same as `spatial_region_gnd` except it has additional context, so why isn't this a refinement?

Reply:
Never mind, the reason is that one `spatial_region_ctx` is derived from the ungrounded version.